Automattic/wp-calypso
# Reader: Remove unnecessary `selectFirstItem` action creator and fix error

#### Changes proposed in this Pull Request

When you go to `/read` (in local environment or `wpcalypso`) and press the dot key `.`, you'll get an error:

![](https://cldup.com/MFLFikMQEA.png)

This is caused by the fact that we're attempting to call the `selectFirstItem` action creator, but it never gets imported or returned as a bound action. 

At first I was going to import it, but then noticed that no reducer is listening to that action. So this PR is effectively removing the wrong call and the unused action creator.

#### Testing instructions

* Go to `/read`
* Press dot (`.`).
* Verify you no longer get an error.
* Verify the removed code is not in use.
